tests/dummy/config/targets.js

```diff
@@ -6,13 +6,6 @@ const browsers = [
   'last 1 Safari versions'
 ];
 
-const isCI = !!process.env.CI;
```

Comment:
Fixes the `Uncaught TypeError: (0 , _emberBabel.extends) is not a function` error.